### PR TITLE
Add fancycompleter recipe

### PR DESCRIPTION
**Pull request by kynan**

### CODE REVIEW

**Comment by kynan** [comment]
Good to go @conda-forge/core?

File: recipes/fancycompleter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - pyreadline  # [win]
+    - pyrepl      # [not win and py2k]
+
+test:
+  imports:
+    - fancycompleter
+
+about:
+  home: https://bitbucket.org/antocuni/fancycompleter
+  license: BSD
```
Comment (ocefpaf) [inline recipes/fancycompleter/meta.yaml]: Is the version known?

Comment (kynan) [inline recipes/fancycompleter/meta.yaml]: No, there's no license file, see comment below

**Comment by jjhelmus** [comment]
Without the attribute line from a BSD license file I'm not sure what rights we (or anyone) has to redistribute the `fancycompleter` package.  I don't believe the author indented this but without a clearly defined attribute the conditions of the BSD license cannot be met.

File: recipes/fancycompleter/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/fancycompleter/meta.yaml]: Many packages don't ship the file but have that info somewhere in the metadata or the docs. If not we need to contact upstream and request it.

Comment (kynan) [inline recipes/fancycompleter/meta.yaml]: I have requested this already, again see comment below.

Comment (jakirkham) [inline recipes/fancycompleter/meta.yaml]: Would add to that issue that the license file should be included in the `MANIFEST.in`.

**Comment by kynan** [comment]
Fair enough. Let's wait and see if there's a reply to the [issue I've raised upstream](https://bitbucket.org/antocuni/fancycompleter/issues/5).

**Comment by jjhelmus** [comment]
Despite my comment, I think it is fine to merge this PR, create a feedstock and make `fancycompleter` available on the conda-forge channel.  I do not believe it was the authors intention to omit the license file. The spirit of the BSD license is followed in this instance.

@kynan Thanks for putting this PR together and for inquiring about the license upstream.

Antonio Cuni, if you happen to stumble upon this, first, thanks you so much for making `fancycompleter` available as open source, your work is really appreciated!  Our concern here is that the [BSD license text](https://opensource.org/licenses/BSD-2-Clause) does not appear to be included in `fancycompleter`.  A common approach to this is to include a file, typically LICENSE or LICENSE.txt, in the project's root directory with this text.  If you have any questions or concerns about this feel free to reach out to me, I would be happy to help in any manner I can.

**Comment by ocefpaf** [comment]
BTW, IMO, it is a bug in the Python classifier logic. If one uses BSD 3-Clause it errors and prevent from uploading to PyPI. Forcing people to use the degenerated `BSD` form in the metadata :-(

**Pull request merged**